File: AudioVisualizer2/app/src/main/java/com/projapps/audiovisualizer/PolicyActivity.java
package com.projapps.audiovisualizer;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.text.Html;
import android.widget.TextView;

public class PolicyActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_policy);
        TextView textView = (TextView) findViewById(R.id.policy_content);
        textView.setText(Html.fromHtml(getString(R.string.policy_content)));
    }
}

File: app/src/main/java/com/projapps/audiovisualizer/PolicyActivity.kt
package com.projapps.audiovisualizer

import android.os.Build
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.text.Html
import android.text.Spanned
import kotlinx.android.synthetic.main.activity_policy.*

class PolicyActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_policy)
        policy_content.text = getString(R.string.policy_content).toSpanned()
    }

    fun String.toSpanned(): Spanned {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
            return Html.fromHtml(this, Html.FROM_HTML_MODE_LEGACY)
        } else {
            @Suppress("DEPRECATION")
            return Html.fromHtml(this)
        }
    }
}
